test/cmdlineTests/gas_test_dispatch_optimize/output

```diff
@@ -2,52 +2,52 @@
 ======= gas_test_dispatch_optimize/input.sol:Large =======
 Gas estimation:
 construction:
-   300 + 259200 = 259500
+   300 + 260400 = 260700
```

Comment:
Why did this one increase? :)

Reply:
This is weird. I analyzed it a little. The thing is that the peephole optimizer is the first thing that is run. This means in the two versions, the CSE step will have a different input. The newly generated code contains many more literal zeros and before this change, there were more DUPi to retrieve the zero from further down the stack.